engines/native/system/user_darwin.go

```diff
@@ -0,0 +1,98 @@
+package system
+
+import (
+	"fmt"
+	"os/exec"
+	"os/user"
+	"strconv"
+	"strings"
+
+	"github.com/taskcluster/slugid-go/slugid"
+)
+
+const defaultShell = "/bin/bash"
+const systemUserAdd = "/usr/sbin/useradd"
```

Comment:
This doesn't exist on my Mac - are you sure this is right?

From memory, on darwin there are different commands for creating users, rather than `useradd`.